inventory/group_vars/k8s-cluster.yml

```diff
@@ -20,7 +20,7 @@ kube_token_dir: "{{ kube_config_dir }}/tokens"
 # This is where to save basic auth file
 kube_users_dir: "{{ kube_config_dir }}/users"
 
-kube_api_anonymous_auth: false
+kube_api_anonymous_auth: true
```

Comment:
Not sure I follow the logic on this with anonymous auth being required. Any details?

Reply:
This is related to RBAC and permitting cert-based auth without a CommonName field. It's not related to insecure http access really.

Reply:
Right, only relates as it gives us a way to perform API healthchecks over https without authentication.